guides/common/modules/ref_examples-ldap-filters.adoc

```diff
@@ -7,6 +7,7 @@ As an administrator, you can create LDAP filters to restrict the access of speci
 [cols="3,9" options="header"]
 |====
 | User | Filter
+| Only User1 |(distinguishedName=cn=User1,cn=Users,dc=domain,dc=example)
```

Comment:
```suggestion
| User1 |(distinguishedName=cn=User1,cn=Users,dc=domain,dc=example)
```

Reply:
The SME might be willing to keep `Only` as he clearly mentioned in the BZ. However, I am open to change if it's technically correct.

Reply:
I believe so. If you have a filter for User1, you expect it only restricts access for User1 and no one else. Thus, `Only` is redundant,

Reply:
Ok :+1: